### PR TITLE
7 add cap organ slim

### PR DESCRIPTION
**Pull request by ubyndr**
Related with #7 

Added cap_organ_slim.owl via robot template.

### CODE REVIEW

File: src/ontology/capo-edit.owl [hunk 1]
```diff
@@ -9,12 +9,11 @@ Prefix(dcterms:=<http://purl.org/dc/terms/>)
```
Comment (ubyndr) [inline src/ontology/capo-edit.owl:9]: Removed pato and ro imports, and added a description, updated license

File: src/ontology/capo-odk.yaml [hunk 2]
```diff
@@ -13,12 +13,9 @@ export_formats:
   - owl
   - obo
   - json
-import_group:
```
Comment (ubyndr) [inline src/ontology/capo-odk.yaml:16]: removed imports

File: src/ontology/capo.Makefile [hunk 3]
```diff
@@ -3,4 +3,14 @@
...
 
+TEMPLATESDIR = ../templates
+
+TEMPLATES=$(patsubst %.tsv, $(TEMPLATESDIR)/%.owl, $(notdir $(wildcard $(TEMPLATESDIR)/*.tsv)))
+
+.PHONY: templates
+templates: $(TEMPLATES)
+
+$(TEMPLATESDIR)/%.owl: $(TEMPLATESDIR)/%.tsv $(SRC)
+	$(ROBOT) -vvv merge -i $(SRC) template --template $< --output $(COMPONENTSDIR)/$*.owl && \
```
Comment (ubyndr) [inline src/ontology/capo.Makefile:14]: removed EFO prefix

File: src/ontology/catalog-v001.xml [hunk 4]
```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
-    <uri name="http://purl.obolibrary.org/obo/capo/imports/ro_import.owl" uri="imports/ro_import.owl"/>
```
Comment (ubyndr) [inline src/ontology/catalog-v001.xml:3]: removed pato and ro imports

File: src/ontology/capo-edit.owl [hunk 5]
```diff
@@ -9,12 +9,11 @@ Prefix(dcterms:=<http://purl.org/dc/terms/>)
...
 Ontology(<http://purl.obolibrary.org/obo/capo.owl>
-Import(<http://purl.obolibrary.org/obo/capo/imports/pato_import.owl>)
-Import(<http://purl.obolibrary.org/obo/capo/imports/ro_import.owl>)
 Import(<http://purl.obolibrary.org/obo/capo/patterns/definitions.owl>)
 Import(<http://purl.obolibrary.org/obo/capo/components/efo_cap.owl>)
-Annotation(dcterms:description "None")
-Annotation(dcterms:license <https://creativecommons.org/licenses/unspecified>)
+Import(<http://purl.obolibrary.org/obo/capo/components/cap_organ_slim.owl>)
+Annotation(dcterms:description "An ontology that contains CAP related ontology terms gathered from different ontologies ")
+Annotation(dcterms:license <http://creativecommons.org/licenses/by/4.0/>)
```
Comment (ubyndr) [inline src/ontology/capo-edit.owl]: removed pato and ro imports, added a description and updated the licence

File: src/ontology/capo-odk.yaml [hunk 6]
```diff
@@ -13,12 +13,9 @@ export_formats:
...
-  products: 
-    - id: ro
-    - id: pato
 robot_java_args: '-Xmx16G'
 components:
   products:
     - filename: efo_cap.owl
+    - filename: cap_organ_slim.owl
```
Comment (ubyndr) [inline src/ontology/capo-odk.yaml:20]: removed imports

File: src/ontology/capo.Makefile [hunk 7]
```diff
@@ -3,4 +3,14 @@
...
+	$(ROBOT) -vvv annotate --input $(COMPONENTSDIR)/$*.owl --ontology-iri $(ONTBASE)/components/$*.owl -o $(COMPONENTSDIR)/$*.owl
```
Comment (ubyndr) [inline src/ontology/capo.Makefile:15]: removed EFO prefix

File: src/ontology/catalog-v001.xml [hunk 8]
```diff
@@ -1,8 +1,7 @@
...
-    <uri name="http://purl.obolibrary.org/obo/capo/imports/pato_import.owl" uri="imports/pato_import.owl"/>
     <uri id="User Entered Import Resolution" name="http://purl.obolibrary.org/obo/capo/patterns/definitions.owl" uri="../patterns/definitions.owl"/>
     <uri name="http://purl.obolibrary.org/obo/capo/components/efo_cap.owl" uri="components/efo_cap.owl"/>
+    <uri name="http://purl.obolibrary.org/obo/capo/components/cap_organ_slim.owl" uri="components/cap_organ_slim.owl"/>
```
Comment (ubyndr) [inline src/ontology/catalog-v001.xml:5]: removed pato and ro imports

**Comment by ubyndr** [comment]
Hi @dosumis, can you please review the parts that I've commented?

File: capo-base.json [hunk 9]
```diff
@@ -137,22 +137,22 @@
           },
           {
             "pred": "http://purl.org/dc/terms/description",
-            "val": "None"
+            "val": "An ontology that contains CAP related ontology terms gathered from different ontologies "
           },
           {
             "pred": "http://purl.org/dc/terms/license",
-            "val": "https://creativecommons.org/licenses/unspecified"
+            "val": "http://creativecommons.org/licenses/by/4.0/"
```
Comment (evanbiederstedt) [inline capo-base.json]: @ubyndr why doesn't https working here?

File: capo-simple.json [hunk 10]
```diff
@@ -6,217 +6,24 @@
         "basicPropertyValues": [
           {
             "pred": "http://purl.org/dc/terms/description",
-            "val": "None"
+            "val": "An ontology that contains CAP related ontology terms gathered from different ontologies "
           },
           {
             "pred": "http://purl.org/dc/terms/license",
-            "val": "https://creativecommons.org/licenses/unspecified"
+            "val": "http://creativecommons.org/licenses/by/4.0/"
```
Comment (evanbiederstedt) [inline capo-simple.json]: Same, why not `https` here?

**Review by evanbiederstedt — APPROVED** [review]
Left two comments

This looks reasonable to me @ubyndr

File: src/ontology/capo-edit.owl (hunk 5 above)
Comment (ubyndr) [reply src/ontology/capo-edit.owl]: Updated license URL to <https://creativecommons.org/licenses/by/4.0/>

File: capo-simple.json (hunk 10 above)
Comment (ubyndr) [reply capo-simple.json]: Took the url from an other ontology edit file, I guess it should have been https instead http. changed it to https.

**Review by dosumis — APPROVED** [review]
Looks good for our current purposes.   I've created at ticket on EFO to request a clean product with assay terms.  We may incorporate in future.

**Pull request merged**